porter/v1/addons.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package porter.v1;
+
+import "porter/v1/porter_app.proto";
+
+enum AddonType {
+  // ADDON_TYPE_UNSPECIFIED is the default value
+  ADDON_TYPE_UNSPECIFIED = 0;
+  // ADDON_TYPE_POSTGRES is the postgres addon type
+  ADDON_TYPE_POSTGRES = 1;
+}
+
+// Addon is the configuration object for tooling or services that can be applied to the cluster alongside porter apps.
+message Addon {
+  // name is the name of the addon
+  string name = 1;
+  // type is the type of type of addon
+  AddonType type = 2;
+  // config is the addon-specific configuration
+  oneof config {
+    // postgres is the configuration for the postgres addon
+    Postgres postgres = 3;
```

Comment:
Curious if Stefan has any thoughts on best practice for numbering... when we add the next config it will be labelled `5`, and that feels a little weird. Not sure if there are other ways to structure this.

Reply:
If you need consecutive numbers, then start those at a later section